apc_sma.c

```diff
@@ -59,6 +59,14 @@ struct sma_header_t {
 #define SMA_RO(sma, i)   ((char*)(sma->segs[i]).roaddr)
 #define SMA_LCK(sma, i)  ((SMA_HDR(sma, i))->sma_lock)
 
+#define SMA_CREATE_LOCK  APC_CREATE_MUTEX
+#define SMA_DESTROY_LOCK APC_DESTROY_MUTEX
+#define SMA_LOCK(sma, i) (APC_MUTEX_LOCK(&SMA_LCK(sma, i)))
```

Comment:
The outer parentheses here are redundant.

Reply:
fixed